### PR TITLE
Subaru: non-obd FW queries logging

### PR DESCRIPTION
**Pull request by jnewb1**
for gen2:
bus 1 -> powertrain ecus (abs, transmission, engine)
bus 0 -> camera

gen1 will likely have everything on bus0

still need a way to get eps, should be on bus 0

### CODE REVIEW

File: selfdrive/car/subaru/values.py [hunk 1]
```diff
@@ -148,6 +148,21 @@ def init_make(self, CP: car.CarParams):
       [SUBARU_VERSION_REQUEST],
       [SUBARU_VERSION_RESPONSE],
     ),
```
Comment (sshane) [inline selfdrive/car/subaru/values.py:150]: whitelist if only for camera?

Comment (jnewb1) [reply selfdrive/car/subaru/values.py:150]: I suspect gen1 will have everything on bus0, not just the camera. Gen2 is where they moved the powertrain to the other bus

Comment (sshane) [reply selfdrive/car/subaru/values.py:150]: This is for the tester present camera fix, isn't it?

File: selfdrive/car/subaru/values.py [hunk 2]
```diff
@@ -148,6 +148,21 @@ def init_make(self, CP: car.CarParams):
...
+
+    # Non-OBD requests
+    Request(
+      [StdQueries.TESTER_PRESENT_REQUEST, SUBARU_VERSION_REQUEST],
+      [StdQueries.TESTER_PRESENT_RESPONSE, SUBARU_VERSION_RESPONSE],
+      bus=0,
+      logging=True
```
Comment (sshane) [inline selfdrive/car/subaru/values.py:157]:
```suggestion
      logging=True,
```

always good to continue commas

File: selfdrive/car/subaru/values.py (hunk 1 above)
Comment (jnewb1) [reply selfdrive/car/subaru/values.py]: ah, yea I'll whitelist it

File: selfdrive/car/tests/test_fw_fingerprint.py [hunk 3]
```diff
@@ -151,10 +151,11 @@ def test_missing_versions_and_configs(self):
   def test_fw_request_ecu_whitelist(self):
     for brand, config in FW_QUERY_CONFIGS.items():
       with self.subTest(brand=brand):
-        whitelisted_ecus = {ecu for r in config.requests for ecu in r.whitelist_ecus}
         brand_ecus = {fw[0] for car_fw in VERSIONS[brand].values() for fw in car_fw}
         brand_ecus |= {ecu[0] for ecu in config.extra_ecus}
 
+        whitelisted_ecus = {ecu for r in config.requests for ecu in (r.whitelist_ecus if len(r.whitelist_ecus) > 0 else brand_ecus)}
+
```
Comment (sshane) [inline selfdrive/car/tests/test_fw_fingerprint.py:158]: I would prefer to explicitly whitelist each request if we introduce them.

**Review by sshane — APPROVED** [review]

**Pull request merged**